ext/libyuv.cmd

```diff
@@ -10,7 +10,7 @@
 git clone --single-branch https://chromium.googlesource.com/libyuv/libyuv
 
 cd libyuv
-git checkout d53f1be
+git checkout 9b17af9b
```

Comment:
We should wait until this revision is used in Chrome. I can write a Chrome CL to update its third_party/libyuv to this revision.

The src/reformat_libyuv.c change in this pull request is fine. Would you like to remove the ext/libyuv.cmd change from this pull request?

Reply:
See https://chromium-review.googlesource.com/c/chromium/src/+/3822634 for the libyuv update in Chrome.

Also please update the following item in CHANGELOG.md:
```
* Update libyuv.cmd: d53f1be (version 1837)
```

Reply:
I have removed the libyuv.cmd change from this PR. I will do it separately once it lands in chromium.